packages/orchestration/src/cosmos-api.ts

```diff
@@ -296,7 +296,7 @@ export interface LiquidStakingMethods {
 /** Methods supported only on Agoric chain accounts */
 export interface LocalAccountMethods extends StakingAccountActions {
   /** deposit payment (from zoe, for example) to the account */
-  deposit: (payment: Payment<'nat'>) => Promise<void>;
+  deposit: (payment: Payment<'nat'>) => Promise<Amount<'nat'>>;
```

Comment:
I see how this change simplifies `depositHandler` but I'm not sure it's a good idea. Let's explore alternatives

Reply:
This is more so updating the types to match the actual return value. We'll get an error in the type guard if we use `M.undefined()` for the return of `LOA.deposit()`

Reply:
ok. I take back my concern anyway. ERTP's purse `.deposit` already does this (returning the deposited amount)